compiler/record-minmax/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+find_package(HDF5 COMPONENTS CXX QUIET)
+
+if(NOT HDF5_FOUND)
+  return()
```

Comment:
(optional) how about showing some warning message that HDF5 was not found and `record-minmax` will not be built.

Reply:
Added a message. Thank you.